ChronoVisor/include/ChronicleMetaDirectory.h

```diff
@@ -19,23 +19,23 @@ class ChronicleMetaDirectory {
 //    std::shared_ptr<std::unordered_map<std::string, Chronicle *>> getChronicleMap() { return chronicleMap_; }
     std::unordered_map<uint64_t, Chronicle *> *getChronicleMap() { return chronicleMap_; }
```

Comment:
Can  you document the data structures and memory layout in a design document?
Eg : Metadatadirectory consists of an unordered map of pointers to chronicle objects (allocated elsewhere in memory)
       Each chronicle object contains an unordered map of pointers to story objects (allocated elsewhere in memory)